proxy/http/remap/unit-tests/plugin_testing_common.h

```diff
@@ -38,6 +38,8 @@
 
 #include "../PluginFactory.h"
 
+const bool DYNAMIC_PLUGIN_RELOAD_IS_ENABLED = true;
```

Comment:
Might as well go all the way to `constexpr`.

Reply:
fixed.